index.html

```diff
@@ -3,7 +3,7 @@
     <!--#include virtual="head.html" -->
     <meta charset="utf-8">
     <meta http-equiv="content-type" content="text/html;charset=utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0">
```

Comment:
If we are limiting the zoom, effectively disabling it, why not just set `user-scalable=no` ?

Reply:
From what I tested, they behaved in the same matter, however, if I'm not mistaken, Safari still allows pinch zooming even if `maximum-scale` or `user-scalable` are set. I went for `maximum-scale` since its description led me to believe it was more accessibility-friendly. I can change it, though, if that's what we want.